dnfdragora/ui.py

```diff
@@ -1061,7 +1062,7 @@ def _setInfoOnWidget(self, pkg) :
         self.info.setValue("")
         if pkg :
             missing = _("Missing information")
-            description = pkg.description.replace("\n", "<br>")
+            description = escape(pkg.description)
```

Comment:
Removing "\n" replacement will fall in one line only description loosing bullet and numbered list for instance.
Could you change this line as 
`
description = escape(pkg.description).replace("\n", "<br>")
`
please?

Reply:
Yes, you're right, I though that escape does that. But no.